cli/command/trust/sign_test.go

```diff
@@ -295,3 +295,13 @@ func TestSignCommandChangeListIsCleanedOnError(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, len(cl.List()), 0)
 }
+
+func TestLocalFlag(t *testing.T) {
```

Comment:
@eiais Not sure I clearly understand that test :thinking: 
Shouldn't we also test *the happy path* (i.e. "Signing and pushing trust data for local image" […]).